doc/70-Troubleshooting.md

```diff
@@ -1,12 +0,0 @@
-# Troubleshooting <a id="troubleshooting"></a>
```

Comment:
Please don't delete the entire file, that's linked from README.md. Just leave the header in there, I'm pretty sure that we will have additional entries there e.g. for the mail from root thingy.